query/src/test/scala/hmda/query/serialization/InstitutionViewProtobufConverterSpec.scala

```diff
@@ -0,0 +1,22 @@
+package hmda.query.serialization
+
+import hmda.model.institution.InstitutionGenerators._
+import hmda.persistence.model.serialization.InstitutionViewEvents.InstitutionViewStateMessage
+import hmda.query.view.institutions.InstitutionView.InstitutionViewState
+import hmda.query.serialization.InstitutionViewProtobufConverter._
+import org.scalacheck.Gen
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class InstitutionViewProtobufConverterSpec extends PropSpec with PropertyChecks with MustMatchers {
+  val institutionList = Gen.listOf(institutionGen)
+
+  property("InstitutionViewState must serialize to protobuf and back") {
+    forAll(institutionList, Gen.choose(0l, 100000l)) { (i, seq) =>
```

Comment:
I think it reads clearer if we use capital L for `Long` literals. This change would make it consistent with the rest of the codebase too